gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/OptionTest.java

```diff
@@ -24,15 +26,37 @@
 
 public class OptionTest {
 
+  private static final BigQueryRpc.Option RPC_OPTION = BigQueryRpc.Option.PAGE_TOKEN;
+  private static final BigQueryRpc.Option ANOTHER_RPC_OPTION = BigQueryRpc.Option.FIELDS;
+  private static final String VALUE = "some value";
+  private static final String OTHER_VALUE = "another value";
+  private static final Option OPTION = new Option(RPC_OPTION, VALUE) {};
+  private static final Option OPTION_EQUALS = new Option(RPC_OPTION, VALUE) {};
+  private static final Option OPTION_NOT_EQUALS1 = new Option(RPC_OPTION, OTHER_VALUE) {};
+  private static final Option OPTION_NOT_EQUALS2 = new Option(ANOTHER_RPC_OPTION, VALUE) {};
+
   @Test
-  public void testOption() {
-    Option option = new Option(BigQueryRpc.Option.PAGE_TOKEN, "token");
-    assertEquals(BigQueryRpc.Option.PAGE_TOKEN, option.rpcOption());
-    assertEquals("token", option.value());
+  public void testEquals() {
+    assertEquals(OPTION, OPTION_EQUALS);
+    assertNotEquals(OPTION, OPTION_NOT_EQUALS1);
+    assertNotEquals(OPTION, OPTION_NOT_EQUALS2);
   }
 
-  @Test(expected = NullPointerException.class)
-  public void testNullRpcOption() {
-    new Option(null, "token");
+  @Test
+  public void testHashCode() {
+    assertEquals(OPTION.hashCode(), OPTION_EQUALS.hashCode());
+  }
+
+  @Test
+  public void testConstructor() {
+    assertEquals(RPC_OPTION, OPTION.rpcOption());
+    assertEquals(VALUE, OPTION.value());
+    try {
+      new Option(null, VALUE) {};
+      fail("Cannot build with empty option.");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [In JUnit4, use the @Test(expected) annotation to denote tests that should throw exceptions](https://www.codacy.com/app/mziccard/gcloud-java/file/2326222220/issues/source?bid=3123103&fileBranchId=3204660#l56)